util/build_configs/cray-internal/setenv-hpe-cray-ex-x86_64.bash

```diff
@@ -326,7 +326,7 @@ else
         list_loaded_modules
     fi
 
-    gen_version_gcc=10.2.0
+    gen_version_gcc=10.3.0
     #[TODO] gen_version_intel=16.0.3.210
     gen_version_cce=10.0.2
```

Comment:
cce 10.0.2 is pretty old at this point. Do we need to look at upgrading this too, or wait until we run into problems?

Reply:
Yeah, let me do that, too. I was fixated on gcc because of the build failure, but the default cce needs to bump as well.